### DISCUSSION TITLE
31313bb1-9741-4515-911e-cc67d081f58b

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9546] [Rules Discussions]
# 31313bb1-9741-4515-911e-cc67d081f58b

Confirm key decisions and action points via email after phone conversations or meetings to ensure a clear record of agreements.

https://www.ssw.com.au/rules/do-you-prepare-then-confirm-conversations-decisions/

**Comment by bradystroud** [comment]
This is a great rule for anyone that gets busy, not just clients.

**Comment by ClaraFang-SSW** [comment]
Check your understanding with the right people first before acting. This keeps everyone aligned, avoids misinformation, and ensures a clear record of decisions and action points.

**Reply by stephanfakossw** [reply]
Great rule! I feel this should be linked to the manage-up rule.
